README.rst

```diff
@@ -44,19 +46,27 @@ indices (e.g. rebuild_index) without having to take downtime or modify code with
 
     $ bin/rake search:initialize
 
-To rebuild a new index without moving the alias and without running catchup, do the following:
+To validate the 'content' alias exists and contains the proper mappings:
 
 .. code-block:: bash
 
-    $ bin/rake search:rebuild_index
+    $ bin/rake search:validate_index
+
+To rebuild the index:
 
 To rebuild a new index from the database and then point the alias 'content' to it, you can use the
 rebuild_index task. This task will also run catchup before and after the alias is moved, to minimize time where the
 alias does not contain all documents.
 
 .. code-block:: bash
 
-    $ bin/rake search:rebuild_index[true]
+    $ bin/rake search:rebuild_index
```

Comment:
Reviewers: when merging the change in the default behavior for rebuild_index, I forgot to update the docs.